scripts/terran/units.lua

```diff
@@ -21,7 +21,7 @@ Load("scripts/terran/unit-terran-marine.lua")
 Load("scripts/terran/unit-terran-science-vessel.lua")
 Load("scripts/terran/unit-terran-scv.lua")
 Load("scripts/terran/unit-terran-siege-tank.lua")
-Load("scripts/terran/unit-terran-vulture.lua")
+--Load("scripts/terran/unit-terran-vulture.lua")
```

Comment:
Could be this line: https://github.com/Wargus/stargus/blob/98f45fa7706d72e4c64dec2ce0fbcbd3db27b627/scripts/terran/unit-terran-vulture.lua#L102C19-L102C19 - since @Jarod42 made it now so that units have to start with `unit-` 😢

Reply:
In fact, there was already requirement that units begin with `"unit-"` at some other places (upgrade/dependencies IIRC).

Reply:
I also didn't see a reason in the references lua file why it didn't work. But to get it "fixed" with low effort I just commented it out.
As long as stargus moving is fundamental broken one unit more helps nobody. 😢

Reply:
>  As long as stargus moving is fundamental broken one unit more helps nobody. 😢

well, vulture actually had the handwritten example code that we want to generate, with acceleration etc. and without larva, we cannot produce zerg units :D

Reply:
Good point to remember me which unit was the good one. So yes, worth a fix ... in a future PR :-P